src/content/smart-contracts/index.md

```diff
@@ -7,19 +7,13 @@ sidebar: true
 
 # Introduction to smart contracts {#introduction-to-smart-contracts}
 
-Smart contracts are the fundamental building blocks of [Ethereum applications](/dapps/). They are computer programs stored on the blockchain that allows us to convert traditional contracts into digital parallels. Smart contracts are very logical - following an if this then that structure. This means they behave exactly as programmed and cannot be changed.
+A smart contract is a digital contract that automatically executes the agreed on terms when its conditions are met. Smart contracts are the fundamental building blocks of [Ethereum applications](/dapps/). Fundamentally, they are computer programs following "if X then Y" logic, which means they behave exactly as programmed and are resistant to participants' reneging on the terms of the contract. 
```

Comment:
I think @setgree's edit reads nicely, and frames the benefits of smart contracts in a context users are already familiar with (i.e., people reneging on contractual agreements). Also, the average reader learning about smart contracts will find the "If X then Y" logic easier to follow (at least, I did at the time).

Reply:
BTW first sentence can be split into to two, as so:
> Smart contracts are digital contracts on the blockchain. When a smart contract's terms are met, the contract automatically executes the outcome that the parties agreed to.

Reply:
I prefer @minimalsm 's original version because it explains what smart contracts are in terms anyone can understand: building blocks, on-chain computer programs. The additional wordage in the newer version is mostly padding - does it really help to say that a "smart contract is a digital contract", for example?

How about the following compromise:

```suggestion
A smart contract is the fundamental building block of [Ethereum applications](/dapps/). Fundamentally, they are computer programs following "if X then Y" logic, which means they behave exactly as programmed and are resistant to participants' reneging on the terms of the contract. 
```

Reply:
I am not the expert here, and I will defer to your judgment; but in general, I think the lede sentence should literally say what the thing is, rather than the function it serves. For instance, the Wikipedia entry on smart contracts begins:

> A smart contract is a computer program or a transaction protocol that is intended to automatically execute, control or document legally relevant events and actions according to the terms of a contract or an agreement.

This seems like it's on the right track. The lede sentence I chose might not be exactly right, but I think that the general goal of beginning with a definition rather than context/implications is the right overall direction.

Just my 2c 😃